Source/WebKit/WebProcess/Plugins/PluginView.h

```diff
@@ -92,8 +93,11 @@ class PluginView final : public WebCore::PluginViewBase {
     void pluginScaleFactorDidChange();
 #if PLATFORM(IOS_FAMILY)
     void pluginDidInstallPDFDocument(double initialScaleFactor);
+    void setSelectionRange(WebCore::FloatPoint pointInRootView, WebCore::TextGranularity);
```

Comment:
Can you add a forward decl for TextGranularity?

Reply:
Will do!